languages/python/exercises/concept/strings/.docs/introduction.md

```diff
@@ -0,0 +1,5 @@
+# Introduction
+
+A `str` typed object in Python is an immutable sequence of Unicode code points. This could include letters, numbers, punctuation, etc. To manipulate strings, Python provides string methods that can transform a string into other types, create new strings based on method arguments, or return information about the string. Python operators are also overloaded to provide the ability to use them with strings, e.g. to add them together with `+`.
```

Comment:
> Python operators are also overloaded to provide the ability to use them with strings, e.g. to add them together with `+`.

This sentence subtly introduces overloading. This might be a little bit early in the track. Would a simpler version also suffice? Something like:

```suggestion
A `str` typed object in Python is an immutable sequence of Unicode code points. This could include letters, numbers, punctuation, etc. To manipulate strings, Python provides string methods that can transform a string into other types, create new strings based on method arguments, or return information about the string. Strings can be concatenated with `+`.
```

Or do you also want to have the student know that `==` and such can be used?